internal/output/output.go

```diff
@@ -511,6 +539,63 @@ func (p PolicyCheckFailures) Error() string {
 	return out.String()
 }
 
+type FinOpsPolicyCheck struct {
+	Failing []FinOpsPolicy
+	Warning []FinOpsPolicy
+	Passing []FinOpsPolicy
+}
+
+func NewFinOpsPolicyChecks(fops []FinOpsPolicy) FinOpsPolicyCheck {
+	fopc := FinOpsPolicyCheck{}
+
+	for _, fop := range fops {
+		if !fop.PrComment {
+			continue
+		}
+
+		if len(fop.Resources) == 0 {
+			fopc.Passing = append(fopc.Passing, fop)
+			continue
+		}
+
+		if fop.BlockPr {
+			fopc.Failing = append(fopc.Failing, fop)
+		} else {
+			fopc.Warning = append(fopc.Warning, fop)
+		}
+	}
+
+	return fopc
+}
+
+func (fopc FinOpsPolicyCheck) Error() string {
+	if len(fopc.Failing) == 0 {
+		return ""
+	}
+
+	out := bytes.NewBuffer([]byte("FinOps policy check failed:\n"))
```

Comment:
```suggestion
	out := &strings.Builder{}
	out := out.WriteString("FinOps policy check failed:\n")
```

strings.Builder is optimized to work with strings